android/src/main/java/io/ably/flutter/plugin/dto/EnrichedConnectionStateChange.java

```diff
@@ -0,0 +1,15 @@
+package io.ably.flutter.plugin.dto;
+
+import io.ably.lib.realtime.ConnectionStateListener;
+
+public class EnrichedConnectionStateChange {
```

Comment:
Ideally, this class should extend `ConnectionStateListener.ConnectionStateChange`